src/components/TagBadge.svelte

```diff
@@ -16,7 +16,19 @@
 
     return 'is-danger';
   }
+
+  function getValue(value) {
+    if (value === true) {
+      return 'yes';
+    }
+
+    if (value === false) {
+      return 'no';
+    }
+
+    return value;
+  }
 </script>
 
 <span class="tag is-dark">{name}</span>
-<span class="tag {getClass(value)}">{value}</span>
+<span class="tag {getClass(value)}">{getValue(value)}</span>

```

Comment:
I'd probably just go change the actual tag values to 'yes' and 'no' and add those to the appropriate class arrays in case we ever actually want true and false values in tags for some other things.

Reply:
Updated with the new solution.